cli/flox-rust-sdk/src/models/manifest.rs

```diff
@@ -583,7 +583,7 @@ pub struct ManifestVariables(
         test,
         proptest(strategy = "proptest_btree_map_alphanum_keys::<String>(10, 3)")
     )]
-    BTreeMap<String, String>,
+    pub(crate) BTreeMap<String, String>,
```

Comment:
nit: i think it's more canonical to keep these newtypes closed and implement (passthru) accessors as needed rather than making the inner type pub which somewhat defeats [the purpose](https://rust-unofficial.github.io/patterns/patterns/behavioural/newtype.html) to begin with.
by way of example, a `ManifestVariables::insert` should fail if the name side has non env friendly characters anything outside of `{a..Z,_}`.